blog/docs/articles/llm-agents-demo.md

```diff
@@ -0,0 +1,136 @@
+# LLM Agents with Knative: A Demo
+
+**Author: [Calum Murray](https://www.linkedin.com/in/calum-ra-murray/), Software Engineering Intern @ Red Hat**
+
+In the previous blog post on LLM Agents, we discussed what LLM Agents are,
+how LLMs call tools, and how Knative can be used to automate the tool calling
+process for LLM Agents. In this blog post we will be looking at a concrete
+example of how you can build an LLM Agent application with Knative handling
+the tool calling and discovery, and explaining what differentiates this
+approach from the other common approaches for building LLM Agents.
+
+## An example LLM Agent using Knative
+
+To showcase the power of building LLM Agents with the tool discovery and
+calling handled by Knative, we are going to build a simple chat application
+that is capable of answering questions about the cost of resources being
+used for a fictitious application we are running. Specifically, in our
+application we are interested in accurately answering questions such as:
+
+> How has the trend in the average resource consumption (CPU, memory)
+changed over the past few months for my application, and what impact
+will this have on my costs?
+
+To begin, we will be deploying the Knative LLM Tool Provider demo into
+a cluster we have access to following the instructions in the README in
+https://github.com/keventmesh/llm-tool-provider. Once we have deployed
+the chat app from this repository, we are able to access it by running:
+
+```sh
+kubectl port-forward svc/chat-app-service 8080:8080
+```
+
+If you access http://localhost:8080 in your browser you will now be able
```

Comment:
The link here is not clickable. If you are not intending make it clickable, then I think adding `` around the link would make more distinct and easier to distinguish with other normal text.